### PR TITLE
Parsing w base structure

### PR DESCRIPTION
**Pull request by theophile-jr**
Feature the basic architecture with a basic parsing for the PGP.

### CODE REVIEW

File: my_pgp
Comment (Thomaltarix) [inline my_pgp:1]: This is the binary file, isn't it ? Maybe not pushing it + putting it into a .gitignore file

File: src/parse.py
Comment (Thomaltarix) [inline src/parse.py:1]: Very ugly but works 😂

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Don't push the binary 😉

File: src/parse.py
Comment (theophile-jr) [reply src/parse.py:1]: Ahah yes just quickstarting the projet as the delivery is in 1week

File: my_pgp
Comment (theophile-jr) [reply my_pgp:1]: ;)

**Review by Thomaltarix — APPROVED** [review]
Great 👍

**Pull request merged**